build-system/release-workflows/upload-release.js

```diff
@@ -100,6 +101,54 @@ function logProgress_(totalFiles, uploadedFiles) {
   }
 }
 
+/**
+ * Compresses a single file with Brotli and writes it to ${path}.br.
+ * @param {string} path
+ * @param {number} sizeHint
+ * @return {Promise<void>}
+ */
+async function brotliCompressFile_(path, sizeHint) {
+  const fileBuffer = fs.readFileSync(path);
+  const compressedBuffer = zlib.brotliCompressSync(fileBuffer, {
```

Comment:
You're creating an series of promises, but doing all operations sync. We could probably speed up throughput by using streams, eg https://gist.github.com/linoyzaga/e992eecfa371014202cbb8e927935234#file-brotlicompressstream-js.

Reply:
Nice! Significantly faster (<1m compared to >3m) - I love streams!!